ioctl/cmd/action/action.go

```diff
@@ -187,15 +191,17 @@ func sendAction(elp action.Envelope, signer string) error {
 			return err
 		}
 		prvKey, err = crypto.HexStringToPrivateKey(prvKeyOrPassword)
-	} else {
+	} else if passwordFlag.Value() == "" {
 		fmt.Printf("Enter password #%s:\n", signer)
 		prvKeyOrPassword, err = util.ReadSecretFromStdin()
 		if err != nil {
 			log.L().Error("failed to get password", zap.Error(err))
 			return err
 		}
-		prvKey, err = account.KsAccountToPrivateKey(signer, prvKeyOrPassword)
 	}
```

Comment:
Is this more straightforward?

```suggestion
	} else {
	    prvKeyOrPassword = passwordFlag.Value().(string)
	}
```